### DISCUSSION TITLE
Works on win 10 but not 11

### DISCUSSION THREAD

**Discussion by 7thwarlord** [KIvoy/nikke-assistant#8] [Q&A]
I could get it working on my windows 10 machine but on windows 11. It can't locate anything tried rescreen shotting them to test if it was that but still not picking up anything, installed the requirements.txt as well still nothing
![Screenshot 2023-11-26 132709](https://github.com/KIvoy/nikke-assistant/assets/42597729/04c4e7f6-02f1-472d-95f6-6e1b1816239c)
[app_log.txt](https://github.com/KIvoy/nikke-assistant/files/13465581/app_log.txt)
[app_error.log](https://github.com/KIvoy/nikke-assistant/files/13465582/app_error.log)

**Comment by KIvoy** [comment]
Thanks for reporting the compatibility issue, unfortunately I have stopped maintaining the repository since May and the old build was developed in a Windows 10 environment. I'm happy to review & accept PRs that can resolve the issue, though in the near future I won't have capacity to look into it myself.
